core/network/source_other.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package network
+
+import (
+	"net"
+
+	"github.com/juju/errors"
+)
+
+type netPackageConfigSource struct{}
```

Comment:
This is named _source_other.go_, because in the next patch it will get a build directive and _source_linux.go_ will be introduced.